docs/framework/data/adonet/ef/provider-manifest-specification.md

```diff
@@ -77,9 +77,9 @@ This section discusses how a data store provider can support the types and funct
  The provider manifest is loaded by the Store Metadata loader (StoreItemCollection), either by using a data store connection or a provider manifest token.  
   
 #### Using a Data Store Connection  
- When the data store connection is available, call DbProvderServices.GetProviderManifestToken to return the token that is passed to the GetProviderManifest method, which returns DbProviderManifest. This method delegates to the provider's implementation of GetDbProviderManifestToken.  
+ When the data store connection is available, call <xref:System.Data.Common.DbProviderServices.GetProviderManifestToken?displayProperty=nameWithType> to return the token that is passed to the <xref:System.Data.Common.GetProviderManifest%2A> method, which returns <xref:System.Data.Common.DbProviderManifest>. This method delegates to the provider's implementation of `GetDbProviderManifestToken`.  
```

Comment:
I's complaining about the new xrefs. Maybe the `%2A` at the end of the second one?